src/librustc_driver/lib.rs

```diff
@@ -111,34 +111,48 @@ const BUG_REPORT_URL: &'static str = "https://github.com/rust-lang/rust/blob/mas
                                       md#bug-reports";
 
 #[inline]
-fn abort_msg(err_count: usize) -> String {
+fn abort_msg(err_count: usize) -> Option<String> {
     match err_count {
-        0 => "aborting with no errors (maybe a bug?)".to_owned(),
-        1 => "aborting due to previous error".to_owned(),
-        e => format!("aborting due to {} previous errors", e),
+        0 => Some("aborting with no errors (maybe a bug?)".to_owned()),
+        1 => None,
+        e => Some(format!("aborting due to {} previous errors", e)),
```

Comment:
an error count can be useful if one is manually going over the results and wants to either (1.) double-check that one has seen them all, or (2.) get an estimate of how many errors one will need to look at...